styles.css

```diff
@@ -336,3 +338,157 @@ footer {
   line-height: 32px;
   color: #42526e;
 }
+
+/*
+Desktop hidden stuff
+*/
+#desktop-header {
+  display: none;
+}
+
+#icons-form-desktop {
+  display: none;
+}
+
+/*
+Desktop Media Query styles
+*/
+@media screen and (min-width: 768px) {
+  #first-container {
+    background-image: url('images/Header-llustration-desktop.png');
+    background-repeat: no-repeat;
+    background-size: 100%;
+  }
+
+  .heading-section {
+    background-image: none;
+  }
+
+  #mobile-header {
+    display: none;
+  }
+
+  #desktop-header {
+    display: block;
+  }
+
+  #desktop-header .nav-list {
+    justify-content: center;
+  }
+
+  #desktop-header .nav-bar-a {
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    margin: 0 10px;
+  }
+
+  #main-heading {
+    font-weight: 800;
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+    color: #172b4d;
+    margin: 24px 0;
+  }
+
+  .heading-section p {
+    line-height: 32px;
+  }
+
+  .works-section {
+    grid-template-columns: 33% 33% 33%;
+    grid-template-areas:
+      "arrow arrow arrow"
+      "header1 header1 header1";
+    align-items: center;
+    justify-content: center;
+  }
+
+  #arrow-btn {
+    grid-area: arrow;
+  }
+
+  .title-card {
+    grid-area: header1;
+  }
+
+  #about-me-article {
+    grid-template-columns: 33% 33% 33%;
+    grid-template-areas:
+      "h2 h2 h2"
+      "articlep articlep articlep"
+      "btn btn btn";
+    align-items: center;
+    justify-content: center;
+    padding-top: 192px;
+    background-image: url('images/about-me-desktop-bg2.png'), url('images/about-me-desktop-bg1.png');
+    background-repeat: no-repeat;
+    background-position:
+      top 380px left,
+      top 50px right;
+  }
+
+  #about-me-article h2 {
+    grid-area: h2;
+  }
+
+  #about-me-article p {
```

Comment:
- [ ] [OPTIONAL] You're paragraph has more width than the provided design, please consider adding width to this ``` p``` tag so it can be like the Figma design

![Screenshot_20220429_172230](https://user-images.githubusercontent.com/25227022/165993586-9eee5cd8-79eb-4ebe-8e81-a3063d2151c3.png)